### ISSUE TITLE
az aks nodepool list does not work well 

### ISSUE THREAD

**Issue by teppeee** [Azure/azure-cli#24185]
## Describe the bug
az aks nodepool list does not work well when the number of nodepool is 10 or more.
(this number is the button to read more appears in azure portal)
It worked when the version was 2.40.0, but not 2.41.0.

**Command Name**
`az aks nodepool list`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
'ContainerServiceClientConfiguration' object has no attribute 'api_version'
Traceback (most recent call last):
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\knack/cli.py", line 233, in invoke
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 663, in execute
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 710, in _run_job
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/core/paging.py", line 129, in __next__
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/core/paging.py", line 76, in __next__
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/mgmt/containerservice/v2022_07_01/operations/_agent_pools_operations.py", line 406, in get_next
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/mgmt/containerservice/v2022_07_01/operations/_agent_pools_operations.py", line 391, in prepare_request
AttributeError: 'ContainerServiceClientConfiguration' object has no attribute 'api_version'

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
create 10 nodepools

## Environment Summary
```
Windows-10-10.0.22621-SP0
Python 3.10.5
Installer: MSI

azure-cli 2.41.0

Extensions:
application-insights 0.1.16
quota 0.1.0

Dependencies:
msal 1.20.0b1
azure-mgmt-resource 21.1.0b1
```
## Additional Context
Kubernetes version 1.24.6

**Comment by yonzhan** [comment]
route to CXP team

**Comment by PramodValavala-MSFT** [comment]
@msyyc Looks like this is the same error as in this issue Azure/azure-sdk-for-python#26506. Would that mean this would require a regeneration of the Container Service SDK?

**Comment by msyyc** [comment]
Hi @PramodValavala-MSFT Could you please try the latest [private package](https://newstorageyyc.blob.core.windows.net/2022-10/sdk_pr_26774/azure_mgmt_containerservice-20.4.0-py3-none-any.whl) which contains fix for the issue. Once it works well, we will release a public version asap

**Comment by PowerSurj** [comment]
for those seeking immediate workaround, look at downgrading az `aks-preview` extension
i did not spend time tracing at which exact release it got broken, but i found it works at below version

```
az extension remove --name aks-preview
az extension add --name aks-preview --version 0.5.94
```

and you don't have to downgrade `azure-cli`

**Comment by PramodValavala-MSFT** [comment]
@msyyc I can confirm with this package, it works. 👍

**Comment by whage** [comment]
> The fix has been merged as part of this PR - #24270. Closing.

Thanks for the quick response and solution. Can we get a new azure-cli debian package with the fix? Maybe I'm mistaken but I can't yet find it. Looking for it here: https://packages.microsoft.com/repos/azure-cli/pool/main/a/azure-cli/
